### ISSUE TITLE
ytdl.getInfo with proxy option not working

### ISSUE THREAD

**Issue by Abhay07** [przemyslawpluta/node-youtube-dl#328]
I'm using ytdl.getInfo method with proxy option
```
const options = ['--proxy', 'http://184.169.230.38:8888']
video = await ytdl.getInfo(videoUrl,options);
console.log(video.formats);
```
The above code logs [].

I'm trying to solve video unavailable issue. Using the same proxy in my browser, i'm able to see the youtube video.

Let me know, how can i fix this.

**Comment by PrasadBroo** [comment]
yeah i also have same issue i think youtube with cipher encrypted  videos are only available for user who requested it in this case we using proxy hence we are not able to view it otherwise videos without chi[pher can viewed over proxy.
